client/verta/verta/_tracking/entity.py

```diff
@@ -132,6 +132,9 @@ def _get_proto_by_name(cls, conn, name, parent):
 
     @classmethod
     def _create(cls, conn, conf, *args, **kwargs):
+        if 'name' in kwargs and kwargs['name'] is None:
+            kwargs['name'] = cls._generate_default_name()
+
```

Comment:
Adding the call to `_generate_default_name()` if `name` as a keyword-arg is `None`.
The trivial changes in this PR are to ensure that `name` is set as a keyword.